packages/@aws-cdk/aws-apigatewayv2/test/http/route.test.ts

```diff
@@ -55,6 +55,25 @@ describe('HttpRoute', () => {
     });
   });
 
+  test('integration is only configured once if multiple routes are configured with it', () => {
```

Comment:
Should we write another test that integrations can be used across HttpApis?

Something like - define 2 stacks, each with an HttpApi that use the same integration and verify that each stack has only one integration resource type.